lib/fake-xhr/index.test.js

```diff
@@ -186,7 +296,16 @@ describe("FakeXMLHttpRequest", function () {
         it("property is set if we support standards CORS", function () {
             assert.equals(sinonFakeXhr.xhr.supportsCORS, "withCredentials" in this.xhr);
         });
+    });
 
+    describe(".timeout", function () {
+        beforeEach(function () {
+            this.xhr = new FakeXMLHttpRequest();
+        });
+
+        it("property is set if we support timeout", function () {
```

Comment:
weird wording, but I see another similar test is above it, so I understand you just stuck to the "standard". try changing it to "should set the property if ...", and if you could fix the other one while you are at it, that would be great :pray: